keys.go

```diff
@@ -548,6 +555,39 @@ func (c *Client) wrap(ctx context.Context, idOrAlias string, plainText []byte, a
 	return pt, ct, nil
 }
 
+// WrapWithKeyVersion function supports KeyVersion Details, PlainText and Cyphertext in response
+func (c *Client) WrapWithKeyVersion(ctx context.Context, idOrAlias string, plainText []byte, additionalAuthData *[]string) (*KeyActionResponse, error) {
```

Comment:
Can you rename the function name as UnwrapV2() to keep it inline with other functions that have 2 versions.